packages/position/src/react/index.js

```diff
@@ -86,7 +86,7 @@ const Position = React.forwardRef((props, forwardedRef) => {
   }, [updateStyle, when])
 
   return (
-    <React.Fragment>
+    <>
```

Comment:
have we used the shorthand fragment syntax before. please double check that it compiles correctly before deployment

Reply:
??? I didn't change that. Weird??? I'll check